src/patterns/SaleHeaderBanner/SaleHeaderBanner.tsx

```diff
@@ -75,41 +81,48 @@ const SaleHeaderBanner = forwardRef<HTMLDivElement, SaleHeaderBannerProps>(
     const { className: baseClassName, ...commonProps } = getCommonProps(props, 'SaleHeaderBanner');
     const isOpenForBidding = auctionState === AuctionStatus.live;
     const isClosed = auctionState === AuctionStatus.past;
+    const showCountdown = isOpenForBidding && auctionEndTime;
+
     return (
-      <div {...commonProps} className={classnames(baseClassName, className)} {...props} ref={ref}>
-        <SeldonImage
-          aspectRatio="16/9"
-          src={imageSrcUrl}
-          alt={String(auctionTitle)}
-          objectFit="cover"
-          className={`${baseClassName}__image`}
-        />
-        <PageMargin className={`${baseClassName}__stack-wrapper`} {...commonProps} {...props} ref={ref}>
-          <div className={`${baseClassName}__stack`}>
-            {isOpenForBidding && children}
-            <Text variant={TextVariants.title1}>{auctionTitle}</Text>
-            <Text variant={TextVariants.string2} className={`${baseClassName}__location`}>
-              {location}
-            </Text>
-            <div className={`${baseClassName}__occurrence-details`}>
-              {occurrenceInformation.map(({ date, occurrenceLabel }) => (
-                <div className={`${baseClassName}__occurrence-details-text`} key={String(date)}>
-                  <Text variant={TextVariants.string2}>{occurrenceLabel}</Text>
-                  <Text variant={TextVariants.string2} className={`${baseClassName}__date`}>
-                    {date}
-                  </Text>
-                </div>
-              ))}
+      <div className={`${baseClassName}__wrapper`}>
+        <div {...commonProps} className={classnames(baseClassName, className)} {...props} ref={ref}>
+          <SeldonImage
+            aspectRatio="16/9"
+            src={imageSrcUrl}
+            alt={String(auctionTitle)}
+            objectFit="cover"
+            className={`${baseClassName}__image`}
+          />
+          <PageMargin className={`${baseClassName}__stack-wrapper`} {...commonProps} {...props} ref={ref}>
+            {showCountdown && (
+              <Countdown endDateTime={new Date(auctionEndTime)} className={`${baseClassName}__countdown`} />
```

Comment:
I believe they moved the countdown timer above the banner image on mobile. Not sure why Figma is showing both above and below. We can utilize the media query util we use in other components like footer.

Reply:
Weird, I think it was only on the top when I first started working on it, but now I see it in both places in figma... But I just pushed up a commit that moves it to the top and gets rid of the bottom border for the mobile layout